core-galleon-pack/src/main/resources/configs/standalone/standalone.xml/config.xml

```diff
@@ -1,5 +1,21 @@
 <?xml version="1.0" ?>
 
 <config xmlns="urn:jboss:galleon:config:1.0" name="standalone.xml" model="standalone">
-    <feature-group name="standalone"/>
+    <layers>
```

Comment:
does order matter?

Reply:
It has no real impact on the generated configuration. The order of layers is the order galleon will process them. Due to dependency (if any), galleon can change this order when generating the config. In addition wildfly persists configuration in a defined order, @bstansberry and yourself know more on that than I do.